server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AuthenticateCountyAdministrator.java

```diff
@@ -54,15 +52,10 @@ public String endpointName() {
    */
   @Override
   public String endpoint(final Request the_request, final Response the_response) {
-    int status = HttpStatus.OK_200;
-    String result = "Authenticated";
-    
+    ok(the_response, "Authenticated");
```

Comment:
Does this "ok" actually set the status, or send anything as a response? (it would help to be able to see the `ok` method, of course). We have to be careful with return values from `endpoint()`, because Spark will just send those out as part of the response; that's why, on the streaming ones, I return `""`.